docs/supported_platforms.md

```diff
@@ -164,3 +164,44 @@ As a workaround we need to create those CRDs depends on OCP version. (this is
 covered in automation)
 OCP 4.6 https://github.com/openshift/csi-external-snapshotter/tree/release-4.6/client/config/crd
 OCP 4.5 https://github.com/openshift/csi-external-snapshotter/tree/release-4.5/config/crd
+
+## Openshift Dedicated
+
+This platform supports deployment of OCP cluster + OCS cluster on top of it.
+
+### Requirements
+
+* Have locally installed OCM CLI, follow [official documentation](https://github.com/openshift-online/ocm-cli).
+* Have locally installed osde2e CLI, follow [official documentation](https://github.com/openshift/osde2e).
```

Comment:
I think this is not needed anymore, right?

Reply:
yes, removed